compose/transfer/launch.sh

```diff
@@ -164,38 +168,50 @@ fi
 # Launching the experiment #
 ############################
 
+function free_resources() {
+  # Wait for games to terminate.
+  # shellcheck disable=SC2046
+  wait $(jobs -p)
+
+  # We need to clean up the networks created by all the many docker-compose calls
+  # or else Docker might later run out of addresses, giving an error:
+  #   ERROR: could not find an available, non-overlapping IPv4 address pool among
+  #   the defaults to assign to the network
+  docker network rm "$(docker network ls | grep "${PROJECT_NAME_PREFIX}" | awk '{print $1;}')"
```

Comment:
Not important as this seems to be working, but would `docker network prune` work here?

Reply:
yeah that'd be a bit more understandable, I just didn't want to remove other Docker networks in case the user cares about them